src/CanopyContext.js

```diff
@@ -60,6 +60,12 @@ export function CanopyProvider({ saplingURL, splinterURL, children }) {
       }
     };
   };
+
+  window.$CANOPY.hideCanopy = () => {
+    const sideNavElement = document.getElementById('root');
+    sideNavElement.classList.add('display-none');
```

Comment:
Could this just directly set the display style inline? i.e `sideNavElement.style.display = 'none'`

Reply:
Could do that but I think it is cleaner to use the classes that canopyjs already provides instead of directly editing the inline css. For this example the difference is trivial but this shows an example of how the classes can be used.